composer.json

```diff
@@ -66,7 +66,11 @@
         "phpstan/phpstan-strict-rules": "^1.5",
         "phpstan/phpstan-symfony": "^1.3",
         "phpstan/phpstan-doctrine": "^1.3",
-        "php-cs-fixer/shim": "^3.50"
+        "php-cs-fixer/shim": "^3.50",
+        "phpunit/phpunit": "^9.5",
```

Comment:
Why not PHPUnit 10. It supports PHP 8.1

Reply:
As far as I can tell, the symfony does not support v10 yet fully: https://github.com/symfony/symfony/issues/49069